atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServices.java

```diff
@@ -82,7 +79,8 @@ static void waitForSchemaVersions(
             // shook hands with goes down, it will have schema version UNREACHABLE; however, if we never shook hands
             // with a node, there will simply be no entry for it in the map. Hence the check for the number of nodes.
             versions = client.describe_schema_versions();
-            if (versions.size() <= 1 && checkNodesAgreement(allowQuorumAgreement, config, versions)) {
+            if (requiredNumberNodesAgreeOnSchemaVersion(allowQuorumAgreement, config, versions)
```

Comment:
This is now quite confusing. We allow a quorum to agree but only if the others are not UNREACHABLE, we only allow 1 to be UNREACHABLE. I Understand how we got here, but maybe we should just make it so that if you have quorum, you can have any number of UNREACHABLE nodes?